### DISCUSSION TITLE
[提问]你们的暑假作业都写完了吗？

### DISCUSSION THREAD

**Discussion by CAONIMA-5418** [share121/inter-knot#2241] [其他]
![20240815190805](https://github.com/user-attachments/assets/dab02dd1-8fc4-47b0-aca2-0f95a95df64e)
你们的暑假作业都写完了吗，还在这里刷绳网？？
没写完的，请容忍我告诉你们，我也没写完...hh
备注：封面来自bilibili主播咕嗷嗷
主页：https://space.bilibili.com/18713309
视频：https://www.bilibili.com/video/BV16Z421T7mY/?spm_id_from=333.999.0.0

**Comment by CAONIMA-5418** [comment]
啊焯！！！图片没整上！！！

**Comment by CAONIMA-5418** [comment]
啊！！！图片又整上了！！！

**Comment by Renalstone** [comment]
啊什么是作业<img src="https://img-static.mihoyo.com/communityweb/upload/befed8e6dd7a9d00918dd257cdd1a71d.png" alt="米游兔糖葫芦" width="75" >

**Comment by Orangered0801** [comment]
初三毕业 没有作业（草还押上韵了<img src="https://img-static.mihoyo.com/communityweb/upload/39f7ce769ba2902dbfee73517f7eec49.png" alt="妮可 奸笑" width="75" >

**Comment by Pinuttt** [comment]
高三毕业 没有作业（敲敲楼上

**Comment by Yukikaze001** [comment]
没有作业捏
<img src="https://img-static.mihoyo.com/communityweb/upload/39f7ce769ba2902dbfee73517f7eec49.png" alt="妮可 奸笑" width="75" >
